deploy/tower-resource-operator.yaml

```diff
@@ -1,6 +1,11 @@
 ---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: tower-operator
+---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```

Comment:
Need to make sure we update the individual component files that are also in the `deploy/` directory and contain some of this content but broken out individually.

Reply:
Updated individual files.